cli/cli_app.go

```diff
@@ -372,6 +375,12 @@ func RunTerragrunt(terragruntOptions *options.TerragruntOptions) error {
 		terragruntOptions.IamRole = terragruntConfig.IamRole
 	}
 
+	// replace default sts duration if set in config
+	defaultIamAssumeRoleDuration := 3600 // Default duration for IAM Assume role session is 3600s (1hr)
+	if terragruntOptions.IamAssumeRoleDuration == int64(defaultIamAssumeRoleDuration) && terragruntConfig.IamAssumeRoleDuration != nil {
```

Comment:
NIT: remove `defaultIamAssumeRoleDuration` and use `options.DEFAULT_IAM_ASSUME_ROLE_DURATION` here instead.

Reply:
Good catch! I hadn't noticed the options package was already imported/used for the purpose of setting consts across the whole project. I've updated this now :)